app/utils/bundle-importer.js

```diff
@@ -97,11 +120,14 @@ YUI.add('bundle-importer', function(Y) {
     _fileReaderOnload: function(file, e) {
       var data;
       var notifications = this.db.notifications;
-      // If the file passed in was a json file. This should only ever be used
+      // We support droping a bundle YAML file and a changeSet JSON file onto
+      // the canvas. The JSON file support should only ever be used
       // for when there is no guiserver is available like in sandbox mode.
-      if (file.name.split('.').pop() === 'json') {
+      var extension = file.name.split('.').pop();
+      var result = e.target.result;
+      if (extension === 'json') {
```

Comment:
I can add that comment however json support is only for people working on the GUI who want to drop an already generated changeSet when the guiserver isn't available.